### PR TITLE
styling footer section icons

### LINKED ISSUE #253: hover effect in follow us section
In follow us section, there are social media icons .I want to add hover effect in icons that gives a smooth interactivity and also in html file there are inline css used I want to change them to external css in  style/css file as it is a good practice so that icons could load faster .
Please assign this issue to me

### PR DESCRIPTION
**Pull request by Muskan-creator**
# Description

Please include a summary of the change and which issue is fixed.
Follow us section is not interactive . I had made changes adding hover effects to social media icons which are adding now great user experience and also change inline css to external css which is responsive.
Fixes  #253 

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## UI /UX changes
   Attach gif or screenshot for changes.
   - Before :
algopr

    - After :
algoprr

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by Muskan-creator** [comment]
@harikesh409 Please review it!
It's my first PR in this project :)

File: index.html [hunk 1]
```diff
@@ -212,17 +212,17 @@ <h5 class="font-weight-bold text-white mb-lg-4 pb-2 margin-top-mobile">Features<
                 </div>
                 <div class="col-lg-3 col-md-3 col-sm-6 col-12">
                     <h5 class="font-weight-bold text-white mb-lg-4 pb-2 margin-top-mobile">Follow Us</h5>
-                    <a href="#" target="_blank"><i class="fa fa-github p-2 text-white" aria-hidden="true"
-                            style="background:#3caea3; border-radius: 50%;"></i>
+                    <a href="#" target="_blank"><i id="follow"class="fa fa-github p-2 " aria-hidden="true"
```
Comment (harikesh409) [inline index.html:215]: According to HTML standards the id should be unique in the page change that to a class and make the name more meaningful like `social-icon`.

Comment (harikesh409) [reply index.html:215]: Add the github repo link as it is already present for Algo Phantoms.

File: team.html [hunk 2]
```diff
@@ -259,17 +259,17 @@ <h5 class="font-weight-bold text-white mb-lg-4 pb-2 margin-top-mobile">Features<
                 </div>
                 <div class="col-lg-3 col-md-3 col-sm-6 col-12">
                     <h5 class="font-weight-bold text-white mb-lg-4 pb-2 margin-top-mobile">Follow Us</h5>
-                    <a href="#" target="_blank"><i class="fa fa-github p-2 text-white" aria-hidden="true"
-                            style="background:#3caea3; border-radius: 50%;"></i>
+                    <a href="#" target="_blank"><i id="follow" class="fa fa-github p-2 " aria-hidden="true"
```
Comment (harikesh409) [inline team.html:262]: Add the github repo link as it is already present for Algo Phantoms.

File: css/style.css [hunk 3]
```diff
@@ -415,7 +415,15 @@ hr {
...
 }
-
+#follow{
+    color:#fff;
+    background-color: #3caea3;
+}
+#follow:hover{
+    color:#17a2b8;
+    background-color: white;
+    border-radius: 50%;
```
Comment (harikesh409) [inline css/style.css:425]: Make the `text-decoration` property to `none` so that on hover the weird blue underline doesn't appear.

**Review by harikesh409 — CHANGES_REQUESTED** [review]

**Comment by harikesh409** [comment]
> @harikesh409 Please review it!
> It's my first PR in this project :)

@Muskan-creator 
Everything looks nice, I have requested minor changes just do those change and push.
Keep up the good work. 🎉

File: team.html (hunk 2 above)
Comment (Muskan-creator) [reply team.html:262]: Which github repo link I didn't understand?

**Comment by Muskan-creator** [comment]
Ok I am doing changes and after I again make a PR

**Comment by harikesh409** [comment]
> Ok I am doing changes and after I again make a PR

We can use this PR only no need to make a new one just push you commits and resolve the conversations.

**Comment by Muskan-creator** [comment]
I have done requested changes please review it @harikesh409

**Review by harikesh409 — APPROVED** [review]

**Review by ravikr126 — APPROVED** [review]

**Review by ravikr126 — CHANGES_REQUESTED** [review]
You can again raise pull request.there is some conflict with your pr

**Comment by Muskan-creator** [comment]
Please merge this @ravikr126 
That pr which i raised again is not showing in my profile

File: index.html [hunk 4]
```diff
@@ -213,22 +217,33 @@ <h5 class="font-weight-bold text-white mb-lg-4 pb-2 margin-top-mobile">Features<
                     </ul>
                 </div>
 
+
+                <div class="col-lg-3 col-md-3 col-sm-6 col-12 socialicons">
+
+                 <div class="col-lg-3 col-md-3 col-sm-6 col-12 socialicons">
```
Comment (harikesh409) [inline index.html:223]: Line 221 and 223 are same

**Review by harikesh409 — CHANGES_REQUESTED** [review]

**Comment by Muskan-creator** [comment]
image
It perfectly works in my local system but I don't why it is not working here.

File: index.html (hunk 4 above)
Comment (Muskan-creator) [reply index.html:223]: It has some conflits i think .
should i again raise pr

**Pull request closed without merging**